src/main/java/mate/academy/security/AuthenticationServiceImpl.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.security;
+
+import mate.academy.exception.AuthenticationException;
+import mate.academy.exception.RegistrationException;
+import mate.academy.model.User;
+
+public class AuthenticationServiceImpl implements AuthenticationService {
+    @Override
+    public User login(String email, String password) throws AuthenticationException {
+        return null;
+    }
+
+    @Override
+    public User register(String email, String password) throws RegistrationException {
+        return null;
```

Comment:
The `register` method is currently returning `null`. Implement the logic to register a new user, ensuring the email is unique and storing the user's information securely. If registration fails, throw a `RegistrationException`.